docs/install-github.md

```diff
@@ -40,69 +40,22 @@
-      pipelines:
-        - name: Frogbot
-          steps:
-            - name: Frogbot_Scan
-              type: Bash # For Windows runner: PowerShell
-              configuration:
-                integrations:
-                  - name: jfrogPlatform
-                  - name: github
-                inputResources:
-                  - name: cron_trigger
-                environmentVariables:
-                  # [Mandatory]
-                  # JFrog platform URL
-                  JF_URL: $int_jfrogPlatform_url
-                  # [Mandatory if JF_USER and JF_PASSWORD are not provided]
-                  # JFrog access token with 'read' permissions for Xray
-                  JF_ACCESS_TOKEN: $int_jfrogPlatform_accessToken
-                  # [Mandatory]
-                  # GitHub personal access token with the following permissions:
-                  # Read and Write access to code, pull requests, security events, and workflows
-                  JF_GIT_TOKEN: $int_github_token
-                  JF_GIT_PROVIDER: "github"
-                  # [Mandatory]
-                  # API endpoint to GitHub Enterprise server
-                  JF_GIT_API_ENDPOINT: $int_github_url
-                  # [Mandatory]
-                  # GitHub organization namespace
-                  JF_GIT_OWNER: ""
-              execution:
-                onExecute:
-                  - curl -fLg "https://releases.jfrog.io/artifactory/frogbot/v2/[RELEASE]/getFrogbot.sh" | sh
-                  - ./frogbot scan-pull-requests
-                  - ./frogbot scan-and-fix-repos
-                  # For Windows runner:
-                  # - iwr https://releases.jfrog.io/artifactory/frogbot/v2/[RELEASE]/frogbot-windows-amd64/frogbot.exe -OutFile .\frogbot.exe
-                  # - .\frogbot.exe scan-pull-requests
-                  # - .\frogbot.exe scan-and-fix-repos
-      ```
-
-      </details>
+   2.5. Create a Pipelines job with one of our [templates](templates/jfrog-pipelines).
 
-   3.6. In the **pipelines.yml**, make sure to set values for all the mandatory variables.
+   2.6. In the **pipelines.yml**, make sure to set values for all the mandatory variables.
 
-   3.7. In the **pipelines.yml**, if you're using a Windows agent, modify the code inside the onExecute sections as described in the template comments.
+   2.7. In the **pipelines.yml
+   **, if you're using a Windows agent, modify the code inside the onExecute sections as described in the template comments.
 
    **Important**
 
-   - For npm, yarn 2, NuGet or .NET: Make sure to set inside the  [frogbot-config.yml](templates/.frogbot/frogbot-config.yml) file the command to download the project dependencies as the value of the **installCommand** variable. For example, `npm i`  or `nuget restore`
+   - For npm, yarn 2, NuGet or .NET: Make sure to set inside the  [frogbot-config.yml](templates/.frogbot/frogbot-config.yml) file the command to download the project dependencies as the value of the
```

Comment:
Does it make sense to have this section here? We'er referring here to the configuration of the `frogbot-config.yml` file, but we already have a dedicated section for the configuration of this file. 
Note that this section is duplicated 4 times throughout the documentation.

Reply:
done